File: C#/ConsoleApplication1/ConsoleApplication1/Start.cs
using System;
using Person;

namespace ConsoleApplication1
{
    class Start
    {
        //Main
        public static void Main()
        {
            Persona x = new Persona();
            Persona y = new Persona("Riccardo", "Golia", 50);

            string s = x.GetFirstName();

            Console.WriteLine(s + "\n");

            Console.WriteLine(x.Name);
            Console.WriteLine(x.Age);
            Console.ReadLine();
        }
    }
}

File: FSP/ControlloreParcheggio/Start.java
public class Start{
	public static void main(String[]args){
		Parcheggio p = new Parcheggio();
		Auto a1 = new Auto(p, 1);
		Auto a2 = new Auto(p, 2);
		Auto a3 = new Auto(p, 3);
		Auto a4 = new Auto(p, 4);
		Auto a5 = new Auto(p, 5);
		Auto a6 = new Auto(p, 6);

		a1.start();
		a2.start();
		a3.start();
		a4.start();
		a5.start();
		a6.start();
	}
}
